CONTRIBUTING.md

```diff
@@ -0,0 +1,223 @@
+# Contributing to nx-cugraph
+
+If you are interested in contributing to nx-cugraph, your contributions will fall
+into three categories:
+1. You want to report a bug, feature request, or documentation issue
+    - File an [issue](https://github.com/rapidsai/nx-cugraph/issues/new/choose)
+    describing what you encountered or what you want to see changed.
+    - Please run and paste the output of the `print_env.sh` script while
+    reporting a bug to gather and report relevant environment details.
+    - The RAPIDS team will evaluate the issues and triage them, scheduling
+    them for a release. If you believe the issue needs priority attention,
+    comment on the issue to notify the team.
+2. You want to propose a new Feature and implement it
+    - Post about your intended feature, and we shall discuss the design and
+    implementation.
+    - Once we agree that the plan looks good, go ahead and implement it, using
+    the [code contributions](#code-contributions) guide below.
+3. You want to implement a feature or bug-fix for an outstanding issue
+    - Follow the [code contributions](#code-contributions) guide below.
+    - If you need more context on a particular issue, please ask and we shall
+    assist.
+
+## Code Contributions
+
+### Your First Issue
+
+1. Read the project's [README.md](https://github.com/rapidsai/nx-cugraph/blob/main/README.md)
+    to learn how to setup the development environment. # FIXME ADD THIS SECTION
+2. Find an issue to work on. [Issues](https://github.com/rapidsai/nx-cugraph/issues)
+    or [help wanted](https://github.com/rapidsai/nx-cugraph/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) labels
+3. Comment on the issue saying you are going to work on it.
+4. Code! Make sure to update unit tests!
+5. When done, [create your pull request](https://github.com/rapidsai/nx-cugraph/compare).
+6. Verify that CI passes all [status checks](https://help.github.com/articles/about-status-checks/), or fix if needed.
+7. Wait for other developers to review your code and update code as needed.
+8. Once reviewed and approved, a RAPIDS developer will merge your pull request.
+
+Remember, if you are unsure about anything, don't hesitate to comment on issues and ask for clarifications!
+
+
+## Building and Installing From Source
+These instructions are tested on supported versions/distributions of Linux,
+CUDA, and Python - See [RAPIDS Getting Started](https://rapids.ai/start.html)
+for the list of supported environments.  Other environments _might be_
+compatible, but are not currently tested.
+
+Prerequisites:
+__CUDA:__
+* CUDA 11.8+
+* NVIDIA GPU, Volta architecture or later, with [compute capability](https://developer.nvidia.com/cuda-gpus) 7.0+
+
+Further details and download links for these prerequisites are available on the
+[RAPIDS System Requirements page](https://docs.rapids.ai/install#system-req).
```

Comment:
I think it might be better to avoid trying to keep up with the often changing RAPIDS-wide requirements for CUDA and instead just direct readers to the RAPIDS docs for that, which always seem to be kept up-to-date:
```suggestion
compatible, but are not currently tested. Further details about requirements are described in the [RAPIDS System Requirements page](https://docs.rapids.ai/install#system-req).
```